examples/whisper/requirements.txt

```diff
@@ -1,7 +1,7 @@
 neural-compressor
 onnx==1.14.0
-onnxruntime>=1.15.0
-onnxruntime-extensions>=0.8.0
+onnxruntime==1.15.0
```

Comment:
Opened a PR into your branch for this https://github.com/microsoft/Olive/pull/585 
Added a check and todo in the prepostprocessing func for version check.